tests/fetch-tests.nix

```diff
@@ -32,7 +32,7 @@ let
       handleEntry =
         name: type:
         let
-          file = "${path}/${name}";
+          file = /${path}/${name};
```

Comment:
Unrelated change. I'll open a separate PR.

What's the motivation? Does this avoid additional copy-to-store actions or something?